save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/ContestDto.kt

```diff
@@ -17,9 +17,9 @@ import kotlinx.serialization.Serializable
 data class ContestDto(
     val name: String,
     @Contextual
-    val startTime: LocalDateTime,
+    val startTime: LocalDateTime?,
```

Comment:
why it can be nullable?

Reply:
Because there is a state when `startTime` is not set yet (actually on frontend when date range is not set)